### ISSUE TITLE
Constant nagging for payment

### ISSUE THREAD

**Issue by Deadlock989** [BiglySoftware/BiglyBT#1276]
See title. Every time I open BiglyBT or queue a new torrent, I'm nagged to pay you.

Please tone this down by about 95%. Thanks.

Java 1.8.0_221 (64 bit)
  Oracle Corporation
c:\program files\java\jre1.8.0_221

SWT v4922, win32, zoom=100, dpi=144
Windows 10 v10.0, amd64 (64 bit)
B2.1.0.0/4 az2

**Comment by parg** [comment]
Not payment, donation. Did you happen to donate? I guess not, otherwise the nag would go away.

If you don't ever plan to donate and expect the team to work without income on some software that you hopefully find useful then you can remove the nag by pretending that you donated by downloading and installing

magnet:?xt=urn:btih:D4DGDKQFYDX7GGNOBOMGGJBTECIXO3TZ&dn=alreadydonated.biglybt

**Comment by Deadlock989** [comment]
Nah, I'll just use some other free software that doesn't constantly guilt-trip me passive-aggressively.

**Comment by parg** [comment]
see yah!

**Comment by NSimonC** [comment]
Java 1.8.0_202 (64 bit)
  Oracle Corporation
c:\program files\biglybt\jre

SWT v4942r22, win32, zoom=100, dpi=96
Windows 10 v10.0, amd64 (64 bit)
B3.0.0.1_B46/4 az3

I have a regular donation by Paypal..the buy a coffee one, and it still nags me. I haven't even reinstalled it, but have updated it regularly and it's bringing the dialog up every open of the software.

I've tried the alreadydonated.bt file you have posted elsewhere, but it's still bringing up the dialog. I'll try the above magnet, and I have noticed there is a config file thing to try.
Kind Regards
